demux/utils/novaseq_samplesheet.py

```diff
@@ -106,7 +123,11 @@ def is_reverse_complement(self) -> bool:
             and self.get_reagent_kit_version() == self.NEW_REAGENT_KIT_VERSION
         )
 
-    def add_dummy_indexes(self, raw_samplesheet) -> "CreateNovaseqSamplesheet":
+    def is_nipt_samplesheet(self) -> bool:
+        """ Determines if a sample sheet if for NIPT demultiplexing, based on the index length in the run paramaters """
+        return self.runparameters.index_reads == NIPT_INDEX_LENGTH
```

Comment:
Does this mean that NovaSeq always use index length 10?

Reply:
That's correctt.